scaffolds/php/README.md

```diff
@@ -0,0 +1,35 @@
+# PHP Scaffold for Code Retreat
+
+## Install PHP tools
+
+You need to install a package manager for PHP called _composer_.
+Execute the following code to install it in this local repository:
+``` bash
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
```

Comment:
Isn't there an ubuntu package this ? Wouldn't it be preferable ?

Reply:
Nop. I never have seen that before.